src/analysis/special_functions/improper_integrals.lean

```diff
@@ -0,0 +1,131 @@
+/-
+Copyright (c) 2023 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+import measure_theory.integral.integral_eq_improper
+import measure_theory.group.integration
+import measure_theory.integral.exp_decay
+import analysis.special_functions.integrals
+
+/-!
+# Evaluation of specific improper integrals
+
+This file contains some integrability results, and evaluations of integrals, over `ℝ` or over
+half-infinite intervals in `ℝ`.
+
+## See also
+
+- `analysis.special_functions.integrals` -- integrals over finite intervals
+- `analysis.special_functions.gaussian` -- integral of `exp (-x ^ 2)`
+- `analysis.special_functions.japanese_bracket` -- integrability of `(1+‖x‖)^(-r)`
+-/
+
+open real set filter measure_theory interval_integral
+open_locale topology
+
+-- TODO: This should probably be moved elsewhere, but where?
+lemma integral_comp_neg_Iic {E : Type*}
+  [normed_add_comm_group E] [normed_space ℝ E] [complete_space E] (c : ℝ) (f : ℝ → E) :
+  ∫ x in Iic c, f (-x) = ∫ x in Ioi (-c), f x :=
+begin
+  have A : measurable_embedding (λ x : ℝ, -x),
+    from (homeomorph.neg ℝ).closed_embedding.measurable_embedding,
+  have := A.set_integral_map f (Ici (-c)),
+  rw measure.map_neg_eq_self (volume : measure ℝ) at this,
+  simp_rw [←integral_Ici_eq_integral_Ioi, this, neg_preimage, preimage_neg_Ici, neg_neg],
+end
```

Comment:
I'm somewhat surprised this can't be solved in a single line, but I'm not familiar enough with this part of the library to golf it.

Reply:
I also thought "there must be a lemma for this", but I couldn't find one, so I wrote it here. (The proof is basically copied from the analogous result for interval integrals, which is [interval_integral.integral_comp_neg](https://leanprover-community.github.io/mathlib_docs/measure_theory/integral/interval_integral.html#interval_integral.integral_comp_neg).)